src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -84,6 +90,23 @@ private void bindHttpServer(Future<Void> startFuture) {
                         startFuture.fail(httpServerAsyncResult.cause());
                     }
                 });
+
+        vertx.setPeriodic(1000, a -> {
+            log.debug("Checking for stale customers {}", timestampMap.size());
+            Iterator it = timestampMap.entrySet().iterator();
+            while (it.hasNext()) {
+                Map.Entry<String, Long> item = (Map.Entry) it.next();
+                if (item.getValue() + this.bridgeConfig.getHttpConfig().getConsumerTimeout() < System.currentTimeMillis()) {
+                    final SinkBridgeEndpoint deleteSinkEndpoint = this.httpBridgeContext.getHttpSinkEndpoints().get(item.getKey());
+                    if (deleteSinkEndpoint != null) {
+                        this.httpBridgeContext.getHttpSinkEndpoints().remove(item.getKey());
+                        deleteSinkEndpoint.close();
+                        log.warn("User {} deleted after timeout {} ms", item.getKey(), this.bridgeConfig.getHttpConfig().getConsumerTimeout());
```

Comment:
It's "consumer", not "User" and actually, the value from the map is not the timeout.

Reply:
I am not using value from the map, but from the configuration.

Reply:
Ah right ... I am blind :-)